src/session/session_api.c

```diff
@@ -271,15 +271,34 @@ __session_close_cached_cursors(WT_SESSION_IMPL *session)
 static int
 __session_close(WT_SESSION *wt_session, const char *config)
 {
+#ifdef HAVE_CALL_LOG
+    WT_CONNECTION_IMPL *conn;
+#endif
     WT_DECL_RET;
     WT_SESSION_IMPL *session;
+#ifdef HAVE_CALL_LOG
+    bool internal_session;
+#endif
 
     session = (WT_SESSION_IMPL *)wt_session;
 
     SESSION_API_CALL_PREPARE_ALLOWED(session, close, config, cfg);
     WT_UNUSED(cfg);
 
+#ifdef HAVE_CALL_LOG
```

Comment:
Please add a comment indicating why this is split as two.

Reply:
@bvpvamsikrishna I have moved the code to `__wt_session_close_internal` function and added a comment as well.